tests/test_deployments.py

```diff
@@ -756,6 +756,22 @@ async def test_run_deployment_with_ephemeral_api(
         assert flow_run.deployment_id == deployment_id
         assert flow_run.state
 
+    async def test_run_deployment_with_deployment_id(
+        self,
+        test_deployment,
+        orion_client,
+    ):
+        d, deployment_id = test_deployment
+
+        flow_run = await run_deployment(
+            f"{deployment_id}",
```

Comment:
Why a string instead of a UUID? Should we test both?

Reply:
We can test both if we can take both! I've adjusted the code to allow for a UUID